rotkehlchen/accounting/accountant.py

```diff
@@ -477,7 +477,9 @@ def _should_ignore_action(
 
         elif action_type == 'ledger_action':
             ledger_action = cast(LedgerAction, action)
-            identifier = ledger_action.identifier
+            # Ignored actions have TEXT type in the database but
+            # ledger actions have int as type for identifier
+            identifier = str(ledger_action.identifier)
```

Comment:
Can you also add a test with 2 ledger actions, ignoring one and not ignoring the other so that we have a regression test for this fix?

Can you also fix the types here? If it always indeed has to be a string returned then the reason mypy did not catch this is because we put it as `Any` in the function return.

Reply:
I checked but everywhere the type is set to `List[str]`. I don't get why mypy doesn't catch it

Reply:
Hm ... no it is not. Same file, same function. Line 458 and the return at 453.

Reply:
I was looking at the moon for light but had a lamp next to me. Thanks for the eyes